bindings/wasm/src/did/wasm_document.rs

```diff
@@ -168,6 +169,35 @@ impl WasmDocument {
     self.0.sign_self(key_pair.0.private(), &method_query).wasm_result()
   }
 
+  /// Signs another DID document using the verification method specified by `method_query`.
+  /// The `method_query` may be the full `DIDUrl` of the method or just its fragment,
+  /// e.g. "#sign-0".
+  ///
+  /// `Document.signSelf` should be used in general, this throws an error if trying to operate
+  /// on the same document. This is intended for signing updates to a document where a sole
+  /// capability invocation method is rotated or replaced entirely.
+  ///
+  /// NOTE: does not validate whether the private key of the given `key_pair` corresponds to the
+  /// verification method. See `Document::verifyDocument`.
```

Comment:
```suggestion
  /// verification method. See {@link Document.verifyDocument}.
```

Reply:
Yeah we should do a doc-comment pass in the Wasm bindings before 0.5. Does `@link` play nicely with markdown when rendered on the wiki?

Reply:
Looks like it's expanded to a relative link in Markdown, which I think works.

`[verifyDocument](#Document.verifyDocument)`